src/ontology/Makefile

```
@@ -197,6 +197,7 @@ GITHUB_ACTION = false
 
 filtered.obo: $(SRC)
 	perl -ne 'print unless (m@^xref: (Orphanet|OMIM|OMIMPS|DOID|EFO|NCIT|SCTID|MESH|UMLS|ICD10CM):@ && !(m@(obsoleteEquivalent|equivalentObsolete|equivalentTo|relatedTo|mondoIsNarrowerThanSource|directSiblingOf|mondoIsBroaderThanSource)@i))' $< | grep -v '^property_value: confidence' | grep -v '^property_value: excluded_subClassOf' | egrep -v '^synonym: .*EXCLUDE' | egrep -v 'relationship: disease_has_basis_in_dysfunction_of (hgnc|HGNC|NCBIGene):' > $@.tmp && mv $@.tmp $@
+	perl -ne 'print if !(/^xref: (Orphanet|OMIM|OMIMPS|DOID):/ && !(/(obsoleteEquivalent|equivalentObsolete|equivalentTo)/i))' $@  > $@.tmp && mv $@.tmp $@
```

Why is `obsoleteEquivalentObsolete` not included in the filter?

---

Very good observation! Thanks. In my views these should not be included in release at all, but for now, to satisfy this issue, lets add it:

```suggestion
	perl -ne 'print if !(/^xref: (Orphanet|OMIM|OMIMPS|DOID):/ && !(/(obsoleteEquivalent|equivalentObsolete|equivalentTo|obsoleteEquivalentObsolete)/i))' $@  > $@.tmp && mv $@.tmp $@
```

Thanks!

---

Ok, I was trying to understand the rationale and didn't see any of these cases (obsoleteEquivalentObsolete) in the initial diff, but asked about it for completeness and try to fill in this knowledge gap.